cypress/integration/publication.spec.js

```diff
@@ -1,20 +1,36 @@
 describe('Publication', () => {
   beforeEach(() => {
-    cy.visit('/');
     cy.server();
+    cy.visit('/publications');
   });
 
-  it('The user should be able to go to the registration page for publication if logged in', () => {
+  it('The user should be able to start registration with a DOI link', () => {
     cy.get('[data-testid=login-button]').click({ force: true });
-    cy.get('[data-testid=menu]').should('be.visible');
-    cy.get('[data-testid=menu]').contains('Test User');
 
     cy.get('[data-testid=new-publication-button]').click({ force: true });
-    cy.url().should('include', '/publications');
+    cy.url().should('include', '/publications/new');
+
+    // Open second panel, and enter DOI link
+    cy.get('.MuiExpansionPanelSummary-content')
```

Comment:
du fikk ikke dette til å fungere med data-testid eller? Kunne satt data-testid til å være "new-publication-link", "new-publication-file", "new-publication-orcid"?

Reply:
Heller det at jeg er litt lei av å sende alle disse testid-ene som props over alt, så jeg prøvde dette. Men det er jo mer sårbart for endringer da :/.
Hva tenker @pcb-unit ?

Reply:
synes ikke vi slipper unna test-ids assa :-)

Reply:
ok, da gjør jeg som Sara sa